app/src/lib/stores/git-store.ts

```diff
@@ -1505,6 +1505,7 @@ export class GitStore extends BaseStore {
 
   public async discardChanges(
     files: ReadonlyArray<WorkingDirectoryFileChange>,
+    repoPath: string,
```

Comment:
Can't we use `this.repository.path` in this method instead of passing? It seems that's what we're doing in the previous version of this code

Reply:
Ah. Good call out. I am so used to working in the app-store where we usually pass repositories into our methods. :P